packages/connect-react/src/components/ControlApp.tsx

```
@@ -0,0 +1,149 @@
+    borderRadius: theme.borderRadius,
+    border: "solid 1px",
+    borderColor: theme.colors.neutral20,
+    color: theme.colors.neutral60,
+    padding: "0.25rem 0.5rem",
+    gridArea: "control",
+  }
+
+  const baseSelectProps: BaseReactSelectProps = {
+    components: { Option: BaseOption },
+    styles: {
+      control: (base: {}) => ({
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)